doc/how_to_contribute/how_to_write_tutorials.rst

```diff
@@ -0,0 +1,59 @@
+How-To Write a Tutorial
```

Comment:
My comments to this section are similar to the comments in "how to write how-to guides." A template tutorial on writing tutorials might help people match the formating you're expecting.

Reply:
see my latest commit and let me know what you think